### PR TITLE
Feat/tour creation

### PR DESCRIPTION
**Pull request by StefTrkulja**
## **Change goal**
As an author,
I want to create tours by specifying the tours name, description, difficulty, and tags,
So that I can organize and manage my tours more effectively from draft status before publishing.
Acceptance Criteria:
1.When creating a tour, the status should automatically be set to draft.
2.The initial price of the tour should be set to 0 by default.
3.The tour should include the following fields:
4.Name (required)
5.Description (required)
6.Difficulty ( easy,intermediate , hard)
7.Tags ( Adventure,Relaxation,Historical,Cultural,Nature)
8.The author should be able to view a list of all tours they have created, including both draft,published and archived tours.
## **projekat_A**

### CODE REVIEW

**Review by nikolinarogan — APPROVED** [review]
All added things seem to work out, tested it and everything compiles and all tests are positive. 
Adding tour feature well done! All neccessary elements and components for tour creation added. Very precise variable and file names. Interface built fine, should be updated later on, when app grows. May recommend usage of angular materials for easier ui design.

File: Explorer/src/app/feature-modules/tour-authoring/addtour/addtour.component.ts
Comment (Vojnic) [inline Explorer/src/app/feature-modules/tour-authoring/addtour/addtour.component.ts:1]: Molim te obriši console logove

**Review by Vojnic — APPROVED** [review]

**Comment by vijukM** [comment]
The newly added features are functioning smoothly, and the tour functionality has been successfully implemented!

File: Explorer/src/app/feature-modules/tour-authoring/addtour/addtour.component.html
Comment (nikolinarogan) [inline Explorer/src/app/feature-modules/tour-authoring/addtour/addtour.component.html:1]: You could use angular built in materials for easier design, however, everything looks very good!

File: Explorer/src/app/feature-modules/tour-authoring/mytours/mytours.component.ts
Comment (nikolinarogan) [inline Explorer/src/app/feature-modules/tour-authoring/mytours/mytours.component.ts:1]: Nice way of handling enums.

**Pull request merged**